user-management-oidc.md

```diff
@@ -96,3 +86,34 @@ If the domain is configured alongside the access token, then requests should con
 The JWT ID token must contain a field (claim) named `hd` with the desired domain value. We verify that the token has been properly signed (using provider certificates) and has not expired before processing the claim.
 
 To remove a previously set Helm value, you will need to set the value to an empty string: `.Values.oidc.hostedDomain = ""`. To validate that the config has been removed, you can check the `/var/configs/oidc/oidc.json` inside the cost-model container.
+
+## Troubleshooting
+
+### Option 1: Inspect all network requests made by browser
+
+Use [your browser's devtools](https://developer.chrome.com/docs/devtools/network/) to observe network requests made between you, your Identity Provider, and your Kubecost. Pay close attention to cookies, and headers.
+
+### Option 2: Review logs, and decode your JWT tokens
+
+```sh
+kubectl logs deploy/kubecost-cost-analyzer
+```
+
+* Search for `oidc` in your logs to follow events
+* Pay attention to any `WRN` related to OIDC
+* Search for `Token Response`, and try decoding both the `access_token` and `id_token` to ensure they are well formed (https://jwt.io/)
+
+### Option 3: Enable debug logs for more granularity on what is failing
+
+[Docs ref](https://github.com/kubecost/cost-analyzer-helm-chart/blob/v1.103/README.md?plain=1#L63-L75)
+
+```yaml
+kubecostModel:
+  extraEnv:
+    - name: LOG_LEVEL
+      value: debug
+```
+
+### Kubecost Support
```

Comment:
H3, needs to be formatted "Kubecost support"

Reply:
Fixed!